edbdeploy/options.py

```diff
@@ -48,7 +48,7 @@ class ReferenceArchitectureOptionDBaaS:
 
 
 class OSOption:
-    choices = ['CentOS7', 'CentOS8', 'RedHat7', 'RedHat8', 'RockyLinux']
+    choices = ['CentOS7', 'CentOS8', 'RedHat7', 'RedHat8', 'RockyLinux8']
     default = 'CentOS8'
     help = textwrap.dedent("""
         Operating system. Allowed values are: CentOS7, CentOS8, RedHat7 and
```

Comment:
Please add RockyLinux8 to the help message.

Reply:
Fixed